src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,85 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
 import mate.academy.service.MovieService;
+import mate.academy.service.impl.CinemaHallService;
+import mate.academy.service.impl.MovieSessionService;
```

Comment:
Similar to the previous comment, `MovieSessionService` is being imported from the `impl` package. Verify that this is the intended interface and that the injector is set up to provide the correct implementation.